components/editor/vars.ftd

```diff
@@ -167,12 +167,8 @@ url: /foo/blog/images/first-image.jpg
 file-name: FASTN.ftd
 language: ftd
 
-customElements.define('cm-editor', CMEditor);
+loading...
```

Comment:
I prefer we add another Boolean in vars and show a proper UI. Create a loading component that hides (using css class, if will destroy the editor, which contains state we want to preserve).

Reply:
I agree but I would suggest we do this in another PR